src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int side;
+
+    public RightTriangle(String color, int side) {
+        super(color);
+        this.side = side;
+        draw();
+    }
+
+    @Override
+    public void draw() {
+        area = this.side * this.side / 2;
```

Comment:
why do we calculate area in the draw method?

Reply:
done